templates/server-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "dctna.fullname" . }}-server
+  labels:
+    {{- include "dctna.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.server.service.type }}
+  ports:
+    - port: {{ .Values.server.service.port }}
+      targetPort: https
```

Comment:
Is the service internally running on HTTPS?

Reply:
yes there is a selfsigned cert.

Reply:
In that case I hope we get it working with the ALB, let's see.